### ISSUE TITLE
Usage of ppx_let gives false positive(?)

### ISSUE THREAD

**Issue by NightBlues** [Kakadu/zanuda#45]
Code like this
```
let%bind res = somefunc ... in
match res with
...
```
triggers `Using 'function' is recommended` alert, though we can't really rewrite this with `function`.

**Comment by Kakadu** [comment]
Yes, it is a kind of false positive. This happens because zanuda analyses typedtree, i.e. the code after macro expansion. This happens also with other syntax extension, AFAIR ppx_show. 

The proper way to fix it would be to lookup for `function` in the parsetree at the same location where it is found in typedtree, but this is not currently implemented.
